src/components/FilteredMoviesPage.js

```diff
@@ -23,13 +29,14 @@ export default function FilteredMoviesPage({ movies, filterByGenres, genres }) {
             Search Page
           </MenuButton>
         </ButtonWrapper>
-        {filteredMovies.map(({ id, title, poster_path, genre_ids }) => (
+        {filteredMovies.map(movie => (
           <Card
-            title={title}
-            image={poster_path}
-            genresMovie={genre_ids}
+            title={movie.title}
+            image={movie.poster_path}
+            genresMovie={movie.genre_ids}
             allGenres={genres}
-            key={id}
+            key={movie.id}
+            onYesWatch={() => onYesWatch(movie, currentUser)}
```

Comment:
I find the replacement of the destructuring assignment in this case is a good solution in benefit of a more readable code when passing the props to _onYesWatch_. 👍🏻 🤓